lib/src/i18n/all.dart

```diff
@@ -28,7 +28,7 @@ const localeMap = <String, FormValidatorLocale>{
   'zh-cn': LocaleZhCN(),
   'en': LocaleEn(),
   'he': LocaleHe(),
-  'jp': LocaleJp(),
+  'ja': LocaleJa(),
```

Comment:
I think just to keep convention let's keep both of them there to provide backwards compatibility. I'll remove invalid one on next v2 release.

```suggestion
  'ja': LocaleJa(),
  'jp': LocaleJa(),
```

Reply:
Agree, I pushed another commit with backward compatibility for jp.